app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt

```diff
@@ -156,14 +149,11 @@ class PatientsTabScreen : BaseScreen<
   private val illustrationLayout
     get() = binding.illustrationLayout
 
-  private val simpleVideoLayout
-    get() = binding.simpleVideoLayout
+  private val simpleIllustrationLayout
+    get() = binding.simpleIllustrationLayout
 
   private val simpleVideoIllustration
```

Comment:
We are still rendering the video illustrations, let's get rid of them. We only want to render the home screen illustrations here.